src/Analysis/GWASResults/Views/Home/HomeTable/ActionsDropdown/ActionsDropdown.jsx

```diff
@@ -31,6 +34,28 @@ const ActionsDropdown = ({ record }) => {
       ),
       disabled: false,
     },
+    {
+      key: '2',
+      label: (
+        <a
+          href=''
+          onClick={(e) => {
+            e.preventDefault();
+            retryWorkflow(
+              record.name,
+              record.uid,
+            ).then(() => {
+              alert('Workflow successfully restarted.');
```

Comment:
Try using an AntD notification instead of a JS alert: 

https://ant.design/components/notification

Example: https://codepen.io/pen?&editors=001

You can set the duration to zero so it requires the user to close it manually if desired

<img width="1001" alt="image" src="https://github.com/uc-cdis/data-portal/assets/113449836/5fe57948-b865-475c-9e0d-db42d2bf5b1d">